_proposals/pup-0001.rst

```diff
@@ -0,0 +1,317 @@
+
+
+Deciding
+^^^^^^^^
+
+* If an obvious consensus arises via voting, the proposal is ratified. This is a bit vague, but it
+  works for the Django community.
+* Discussion on the thread can confirm if consensus is obvious or not.
+* Votes come from anyone, not just core devs
+* A -1 from a core dev is considered a veto vote and will block ratification until the vote is
+  recast.
+* Core devs are expected to consider and echo serious feedback from the community especially any -1
+  votes from community members.
+* Consensus is not always possible. If obvious consensus is not reached, then the core devs decide.
+
+
+Proposal Requirements
+=====================
+
+This is a modified version of [the Foreman template](https://github.com/theforeman/rfcs/blob/master/0000-template.md).
+
+::
+
+    ---
+    title: <title goes here>
+    author: <author or authors named>
+    ---
+
+    Summary
+    =======
+
+    One paragraph explanation of the feature.
+
+
+    Motivation
+    ==========
+
+    Why are we doing this? What problem is it designed to solve? What is the expected outcome?
+
+
+    Detailed Design
+    ===============
+
+    This is the bulk of the RFC. Explain the design in enough detail for somebody familiar with the
+    project to understand, and for somebody familiar with the codebase to implement. This should get
+    into specifics and corner cases, and include examples of how the feature is used.
+
+
+    Drawbacks
+    =========
+
+    Why should we not do this?
+
+
+    Alternatives
+    ============
+
+    What other designs have been considered? What is the impact of not doing this?
+
+
+    Unresolved Questions
+    ====================
+
+    What parts of the design are still TBD?
+
+
+Reaching a Decision
+===================
+
+How Long?
+^^^^^^^^^
+
+The initial announcement needs to identify a must-respond-by date and time. This should be no less
+than 10 calendar days from the initial announcement. Longer periods are also OK, and can be adjusted
+via the list based on feedback via the list. I think using calendar days is better than business
+days because the whole world treats calendar days the same. I suggest 10 because sometimes people go
+offline for a whole week. Longer than 10 days and I think the process could become too slow.
+
+
+Why Approve/Reject This Way?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* It treats community decision making (not just core dev decision making) as a first class principle
+* It works for the best communities we know of (Django, Python, Apache, etc)
+* It allows people to be as involved or not as they way
+* It allows serious concerns to not be dismissed (due to a -1 veto vote from core dev)
+
+
+Approval
+^^^^^^^^
+
+* The proposal index is updated moving it from an active section to an approved area. This is
+  similar to the index of PEPs for Python.
+* A post-approval announcement should be sent in the same thread as the initial announcement and
```

Comment:
A new thread for the announcement would be valuable. Most readers are likely to eventually tune out a long discussion about details of a proposal they don't feel strongly about. They could easily miss the announcement if it's part of that thread.